### PR TITLE
Fix naming

### PR DESCRIPTION
**Pull request by metalurgical**
renames tkey_pkg to tkey-mpc-swift

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -4,20 +4,20 @@
...
-            name: "ThresholdKey",
-            targets: ["tkey-pkg"]),
+            name: "tkey-mpc-swift",
+            targets: ["tkey-mpc-swift"]),
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
-        .package(name: "TorusUtils", url: "https://github.com/torusresearch/torus-utils-swift" , from: "6.1.0"),
+        .package(name: "TorusUtils", url: "https://github.com/torusresearch/torus-utils-swift" , .branch("extension")),
```
Comment (metalurgical) [inline Package.swift]: This needs to be updated to a version when https://github.com/torusresearch/torus-utils-swift/pull/79 merges

**Review by himanshuchawla009 — APPROVED** [review]
lgtm

**Review by himanshuchawla009 — APPROVED** [review]
lgtm

**Review by ieow — APPROVED** [review]
lgtm

**Pull request merged**